src/Squidex.Domain.Apps.Core.Model/Contents/StatusChange.cs

```diff
@@ -1,17 +1,16 @@
 ﻿// ==========================================================================
 //  Squidex Headless CMS
 // ==========================================================================
-//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  Copyright (c) Squidex UG (haftungsbeschränkt)
 //  All rights reserved. Licensed under the MIT license.
 // ==========================================================================
 
 namespace Squidex.Domain.Apps.Core.Contents
 {
     public enum StatusChange
     {
-        Archived,
```

Comment:
why have we removed Archived status?

Reply:
It is not the status. It is the transition from Draft => Archived or from Published to Archived.

But when we generalize the workflow we cannot assume anymore that there is a new Archived state, so I have removed this. Published is a special case, it must always exist.